logging/syslog.go

```diff
@@ -34,7 +34,7 @@ func (s *SyslogWrapper) Write(p []byte) (int, error) {
 		y := bytes.IndexByte(p[x:], ']')
 		if y >= 0 {
 			level = string(p[x+1 : x+y])
-			afterLevel = p[x+y+2:]
+			afterLevel = bytes.TrimLeft(p[x+y+2:], " ")
```

Comment:
I'm not sure we need this. I'm curious if you were seeing messages with extra spaces between the level generated somewhere? If so, could you tell us where? 

I made sure the test cases covered this little bit, so if you decide to remove it, you'll need to update the tests to remove those extra spaces.

Reply:
Nope, I was seeing messages without extra spaces between the level generated. I was seeing messages with extra spaces before the log message text (an MSG field RFC3164/5424). In other words, all messages contain MSG field with preceding extra space.